src/number.cr

```diff
@@ -4,10 +4,40 @@ struct Number
 
   alias Primitive = Int::Primitive | Float::Primitive
 
+  # Returns the value zero in the respective type.
+  #
+  # ```cr
```

Comment:
```suggestion
  # ```
```

Reply:
I don't see the point. Omitting the language is fine because the context suggest it's crystal, but being explicit about it isn't wrong.

Reply:
Well, it breaks the formatter, so I'd say that's pretty wrong.

Reply:
Also, doc generator doesn't support language markers whatsoever.

Reply:
Oh, I didn't realize it's broken. I'll change it then. But we should definitely fix the formatter/doc generator. Or have the formatter remove the language tag.

Reply:
IIRC language tags are used for _anything else_ than Crystal code.